### PR TITLE
Export DICOM images from AltaDB

### PR DESCRIPTION
**Pull request by AvanishCodes**
Command
---------
`altadb export <datasetname> <path>`

Validations
---------
- [X] Tested with DICOM image of multiple `SeriesInstanceUID`.
- [X] Tested with dataset of images more than number of images received in Single page response.
- [ ] Tested with RGB DICOM Image.
- [ ] Tested for Diff modality images.
- - [ ] CT
- - [ ] MR

### CODE REVIEW

**Review by pritamrungta — CHANGES_REQUESTED** [review]

File: altadb/utils/files.py [hunk 1]
```diff
@@ -317,3 +322,31 @@ async def _download_file(
...
     return [(path if isinstance(path, str) else None) for path in paths]
+
+
+async def get_image_content(aiosession: aiohttp.ClientSession, image_url: str) -> bytes:
+    """Get image content."""
+    async with aiosession.get(image_url) as response:
+        return await response.content.read()
+
+
+def create_dicom_dataset(
```
[the remark this answers is not in the document]
Comment (AvanishCodes) [reply altadb/utils/files.py:333]: Is it fine to take the `aiohttp.ClientSession` object as the input too?

File: altadb/export/public.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        first_iteration: bool = True
+        end_cursor: Optional[str] = None
+        dataset_root = f"{path}/{dataset_name}"
+        console = Console()
+
+        console.print(f"[bold green][\u2713] Saving dataset {dataset_name} to {path}")
+
+        json_path = f"{dataset_root}/series.json"
+
+        if not ignore_existing:
```
[the remark this answers is not in the document]
Comment (AvanishCodes) [reply altadb/export/public.py:87]: To not download a series if it exists.

File: altadb/cli/command/query.py [hunk 3]
```diff
@@ -44,11 +44,22 @@ def handle_query(self) -> None:
...
-        )
+        first_iteration: bool = True
+        end_cursor: Optional[str] = None
+        status: List[Dict[str, str]] = []
+        while first_iteration or end_cursor:
+            entries, end_cursor = (  # pylint: disable=unused-variable
+                self.cli_dataset.context.dataset.get_data_store_imports(
+                    org_id=cli_dataset.creds.org_id,
+                    data_store=dataset,
+                    first=number,
```
[the remark this answers is not in the document]
Comment (AvanishCodes) [reply altadb/cli/command/query.py]: Use `PaginatedResponse`.

File: altadb/export/public.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (AvanishCodes) [reply altadb/export/public.py]: Removing this feat.

File: altadb/utils/files.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AvanishCodes) [reply altadb/utils/files.py]: Not taking `aiohttp.ClientSession` as in input parameter.

**Comment by pritamrungta** [comment]
merging without approval

**Pull request merged**